### DISCUSSION TITLE
快捷键ctrl+,经常不能正常工作

### DISCUSSION THREAD

**Discussion by colinzhaoxp** [tamlok/vnote#1778] [Show and tell]
如题，每当启动软件后还可以正常使用，但是简单编辑完几句话后就该快捷键就无法正常工作了。使用的vnote版本是3.1.0

**Comment by tamlok** [comment]
系统？是否vi？一直英文状态下输入就突然不行？其他软件？

具体一点？

**Reply by colinzhaoxp** [reply]
抱歉没有提供详细的信息。

系统是win10家庭中文版，版本号20H2

关于是否vi，第一次接触vnote，有点小白，3.1.0版本怎么触发vi模型呢？一直没找到相应的文档说明，只看到说有vi模式。

中英文模式下，其他快捷键都是可以的。我也尝试关闭其他软件，但是没有解决问题，也尝试用过快捷键的检测，并没有冲突。

每次启动后可以正常编辑使用ctrl+，快捷键，编辑完几句后这个快捷键就无法使用了，好像ctrl+,和ctrl+.有冲突。

情况可能不好复现，麻烦作者了。

**Reply by mudssky** [reply]
vi模式在设置，文本编辑器，输入模式里面选

**Comment by tamlok** [comment]
https://github.com/vnotex/vnote/issues/1781
